packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/categories/UpdateCategoriesTests.java

```diff
@@ -93,6 +94,27 @@ public void testUpdateSubcategory_asAdmin()
         updatedCategory.assertThat().field(FIELD_NAME).is(categoryNewName);
     }
 
+    /**
+     * Try to update a category with a name, which is already present within the parent category
+     */
+    @Test(groups = { TestGroup.REST_API})
+    public void testUpdateCategory_usingRecurringName()
```

Comment:
This looks like a good extra test, but it seems unrelated to the change. I don't have an objection - more tests is good as you think of them, I just wanted to check that you intended to include this.